org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/vcs/VcsFacade.java

```diff
@@ -39,4 +39,8 @@ default void addHeadRefsChangeListener(Consumer<List<ISonarLintProject>> listene
 
   }
 
+  default void removeHeadRefsChangeListener() {
+
```

Comment:
Sad empty line noises 😢

Reply:
Any suggestion?

Reply:
No, I did not see that is used in `NoOpVcsFacade` as well at first and was wondering why it was not part of `AbstractEGitVcsFacade`. That's my bad.